src/index.html

```diff
@@ -17,6 +17,83 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
```

Comment:
Consider adding a descriptive text for the link, such as `aria-label`, to improve accessibility for screen readers.